src/Scripting/TasksDate.ts

```diff
@@ -109,4 +110,18 @@ export class TasksDate {
         const unit = words[1] as DurationInputArg2; // day, days, weeks, month, year
         return earlier ? now.subtract(multiplier, unit) : now.add(multiplier, unit);
     }
+
+    public postpone(unitOfTime: unitOfTime.DurationConstructor = 'days', amount: number = 1) {
+        if (!this._date) throw new Notice('Cannot postpone a null date');
```

Comment:
For an example alternative approach, see `bulkAddStatusCollection()` - which returns a container of strings, one for each notice that will need to be displayed.